semcore/switch/src/index.d.ts

```diff
@@ -43,7 +43,14 @@ export type SwitchValueProps = BoxProps &
 
 /** @deprecated */
 export interface ISwitchAddonProps extends SwitchAddonProps, UnknownProperties {}
-export type SwitchAddonProps = BoxProps & NeighborItemProps & {};
+export type SwitchAddonProps = BoxProps &
+  NeighborItemProps & {
+    /** @deprecated
```

Comment:
Also this JSDoc tag is not parsing correctly
```
    /**
     * @deprecated
 ```
 
 Will work

Reply:
both options are correct